packages/arb-token-bridge-ui/src/components/common/Dialog.tsx

```diff
@@ -128,13 +128,15 @@ export function Dialog(props: DialogProps) {
                   variant="secondary"
                   onClick={() => props.onClose(false)}
                   {...(props.cancelButtonProps || {})}
+                  aria-label="Cancel"
```

Comment:
NIT : Cancel is a very generic label and can be detected by tests in other locations in the app as well. To avoid conflicts, can we be more verbose with these labels?